imagemounter/filesystems.py

```diff
@@ -636,14 +636,18 @@ def mount(self):
         os.environ['LVM_SUPPRESS_FD_WARNINGS'] = '1'
 
         # find free loopback device
-        self._find_loopback()
+        # No need to find loopback device if using nbd device
+        if "nbd" not in self.volume.get_raw_path():
+            self._find_loopback()
+        else:
+            self.loopback = ""
```

Comment:
I'd prefer to set it to None, in other words, to just remove the else statement here.